### PR TITLE
Restaurant Page pull request

### PR DESCRIPTION
**Pull request by Grifo89**
# This pull request has:
- A restaurant page made mostly based on javascript.

### CODE REVIEW

**Review by MauricioRobayo — CHANGES_REQUESTED** [review]
Congratulations, very nice project.

I like the structure that you have and the way you handled your SCSS partials.

There is a requirement that is not fully met:

* [x] It features a tabbed menu. When a tab is clicked the content is replaced ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/tab_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/restaurant-page)). Specifically, check the tab-switching logic inside of `ìndex.js`. You should have event listeners for each tab that wipes out the current contents and then runs the correct `tab module` to populate it again.

Status: __Request changes__.

**Comment by Grifo89** [comment]
Changes suggested are done.

File: src/index.js [hunk 1]
```diff
@@ -9,7 +9,28 @@ const content = document.getElementById('content');
...
+      content.appendChild(Menu.List);
+      e.preventDefault();
+    });
+  } else if (item.textContent === 'Booking') {
+    item.addEventListener('click', (e) => {
+      nav.nextSibling.remove();
+      content.appendChild(Booking);
+      e.preventDefault();
+    });
+  }
```
Comment (shloch) [inline src/index.js:35]: great job

**Review by shloch — APPROVED** [review]
accomplished
### great job

**Pull request merged**